src/Datagrid/DatagridInterface.php

```diff
@@ -23,90 +23,66 @@
  */
 interface DatagridInterface
 {
-    /**
-     * @return PagerInterface
-     */
-    public function getPager();
+    public function getPager(): PagerInterface;
 
-    /**
-     * @return ProxyQueryInterface
-     */
-    public function getQuery();
+    public function getQuery(): ProxyQueryInterface;
 
     /**
-     * @return array
+     * @return object[]
      */
-    public function getResults();
+    public function getResults(): array;
 
-    public function buildPager();
+    public function buildPager(): void;
 
-    /**
-     * @return FilterInterface
-     */
-    public function addFilter(FilterInterface $filter);
+    public function addFilter(FilterInterface $filter): void;
```

Comment:
Should return `FilterInterface`? I haven't checked, just based on the comment.

Reply:
Nope, this was just changed on 3.x when bumping phpstan to level 3, the correct type is void based on the implementation.